specs/models/contracts/ExchangeContractModel.spec.js

```diff
@@ -44,4 +44,11 @@ describe('exchange contract model', () => {
       'sellPrice': null
     })
   })
+
+  it('should not validate', () => {
+    const values = new Map()
+    values.set('buyPrice', 20)
+    values.set('sellPrice', 10)
+    expect(validate(values).sellPrice.length).toEqual(1)
```

Comment:
expect(validate(values).sellPrice.length).toEqual(1) > expect(validate(values).sellPrice).not.toEqual(null)